src/index.css

```diff
@@ -55,7 +54,7 @@ code {
 }
 
 .bottombar-design{
-  background-color: #191919;
+  background-color: transparent;
   padding-top: 20px !important;
```

Comment:
please try to avoid this, you can achieve same outcome by linking classes like
.MY_PARENT .MY_CHILD { ..... }